package.json

```diff
@@ -16,16 +17,23 @@
     "bluebird": "^3.0.6",
     "lodash": "^4.0.0"
   },
-  "optionalDependencies": {
-    "bcrypt": "~0.8.0",
-    "bcryptjs": "^2.3.0"
-  },
   "devDependencies": {
+    "@types/bluebird": "^3.5.5",
```

Comment:
These types dependencies should be in `dependencies` rather than `devDependencies` so that users of this module have them available for typescript to make use of